### ISSUE TITLE
Allow connection to a remote daemon

### ISSUE THREAD

**Issue by sdesbure** [calxibe/StorjMonitor#19]
today in the code (https://github.com/calxibe/StorjMonitor/blob/master/storjMonitor.js#L29), the connection assumes that:

* the daemon is on the same host
* the port is the default one

As I'm using [oreandawe/strojshare-cli](https://hub.docker.com/r/oreandawe/storjshare-cli/) container, the daemon is not on the same host.

Therefore, it would be good to make variables like for token:
```javascript
var token = "YOUR-TOKEN-HERE"; //api token can be create under "Nodes" -> "API-Key"
var daemon_host= "127.0.0.1"; //where the storj daemon is running
var daemon_port= 45015; //on which port storj daemon is listening
var log = console.log;
...
function fParseNodes() {
	const daemon = dnode.connect(daemon_host, daemon_port);
...
```

it would also be awesome to have the configuration in a json file (for example `config.json`):
```json
{
  "token": "YOUR-TOKEN-HERE",
  "daemon_host": "127.0.0.1",
  "daemon_port": 45015
}
```

then in the code:
```javascript
var config= require('./config.json')
```
and then use `config.token`, `config.daemon_host` and `config.daemon_port`.

The interest (for Docker user) is the ability to mount the config (with `--volume`) and then being able to upgrade easily :)

If you're interested with that, I can make 3 pull requests:

- the first, just adding the two new variables
- the second, moving variables to a `config.json` and handling them
- the third, adding a Dockerfile that would build automatically with new features (I already have one ready that works today)

**Comment by stefanbenten** [comment]
Already merged your first pull request #20 👍 
Feel free to implement the other solutions as well.
I will rework the install routine in the meantime!

**Comment by sdesbure** [comment]
I've proposed a pull request for the Dockerfile. I'll see later for the "config.json" part :)
